style.css

```diff
@@ -0,0 +1,762 @@
+  background-color: white;
+  margin: 20px 0 0 20px;
+  padding: 15px 20px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+}
+
+.btn-5:hover {
+  background-color: #396df2;
+  color: white;
+}
+
+.myself .lang {
+  width: 90%;
+  margin: 0 auto;
+  list-style: none;
+}
+
+.myself .lang .lang-des li {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  padding: 25px 0;
+  margin-top: 30px;
+}
+
+.myself .lang .lang-des li p {
+  color: #000;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+}
+
+.myself .lang .lang-des li img {
+  margin-right: 20px;
+}
+
+.myself .lang .frame li img {
+  margin-right: 20px;
+}
+
+.myself .lang .skills li img {
+  margin-right: 20px;
+}
+
+.myself .lang .know .java {
+  display: flex;
+  flex-flow: row wrap;
+  gap: 8px;
+  border-radius: 8px;
+  padding: 10px 0;
+  background-color: #f7f7f9;
+}
+
+.myself .lang .know .java img {
+  margin-left: 5px;
+}
+
+.myself .lang .frame li p {
+  color: #000;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+}
+
+.myself .lang .skills li p {
+  color: #000;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+}
+
+.myself .lang .know .java p {
+  color: #253858;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 20px;
+  margin-top: 18px;
+}
+
+.myself .lang .know .htmll {
+  display: flex;
+  flex-flow: row wrap;
+  gap: 8px;
+  border-radius: 8px;
+  padding: 10px 0;
+  background-color: #f7f7f9;
+  margin-top: 10px;
+}
+
+.myself .lang .know .htmll img {
+  margin-left: 5px;
+}
+
+.myself .lang .know .htmll p {
+  color: #253858;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 20px;
+  margin-top: 18px;
+}
+
+.myself .lang .know .csss {
+  display: flex;
+  flex-flow: row wrap;
+  gap: 8px;
+  border-radius: 8px;
+  padding: 10px 0;
+  background-color: #f7f7f9;
+  margin-top: 10px;
+}
+
+.myself .lang .know .csss img {
+  margin-left: 5px;
+}
+
+.myself .lang .know .csss p {
+  color: #253858;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 20px;
+  margin-top: 18px;
+}
+
+.myself .lang .frame li {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  padding: 25px 0;
+}
+
+.myself .lang .skills li {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  padding: 25px 0;
+}
+
+.myself .lang .line {
+  width: 100%;
+  height: 1px;
+  background-color: #dfe1e6;
+}
+
+/* Contact Form */
+.co-form {
+  width: 100%;
+  height: auto;
+  background-image: url("image/contact-form-bg.png");
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 50px 0 45px 0;
+  border-radius: 60px 0 0 0;
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: center;
+  margin-top: 114px;
+}
```

Comment:
- Hey, let's also use the correct background image for the desktop version of the contact form to make your design more consistent following the original design as much as you can. see the images below:


*Your design*
![image](https://github.com/mar4444/re-portfolio-de/assets/59397159/e5689d95-35b3-429b-8fd6-adad1ba9f7f2)



*Figma design*
![image](https://github.com/mar4444/re-portfolio-de/assets/59397159/06679a79-af2d-44da-a4b4-c36daaa96fe1)